infrastructure-playbooks/shrink-osd.yml

```diff
@@ -117,6 +117,16 @@
         _lvm_list: "{{ _lvm_list | default({}) | combine(item.stdout | from_json) }}"
       with_items: "{{ _lvm_list_data.results }}"
 
+    - name: Refresh /etc/ceph/osd files
+      ceph_volume_simple_scan:
+        cluster: "{{ cluster }}"
+        force: true
+      environment:
+        CEPH_CONTAINER_IMAGE: "{{ ceph_docker_registry + '/' + ceph_docker_image + ':' + ceph_docker_image_tag if containerized_deployment | bool else None }}"
+        CEPH_CONTAINER_BINARY: "{{ container_binary }}"
+      delegate_to: "{{ item.0 }}"
+      loop: "{{ _osd_hosts }}"
```

Comment:
When passing a list of osd ids, (`-e osd_to_kill=1,3,5`), if the osds are on the same host, that command would be run multiple times unnecessarily on that host.